artemis-kryo/src/main/java/com/artemis/io/KryoArtemisSerializer.java

```diff
@@ -66,27 +63,14 @@ public void setKryoInstantiatorStrategy (InstantiatorStrategy strategy) {
 	@Override
 	protected void save(Writer writer, SaveFileFormat save) {
```

Comment:
lets just throw an `UnsupportedOperationException` and `@Deprecate` the method. The default case ought to be an Input/OutputStream for load/save.